content/guides/getting-started/component-testing.md

```diff
@@ -42,7 +42,7 @@ website. This results in tests that are faster and have fewer dependencies on
 infrastructure than end-to-end tests that cover the same code paths.
 
 Lastly, Cypress's API is user-centric and built for testing things that render
-in the web. Therefor, many of your tests will appear framework-agnostic and
+in the web. Therefore, many of your tests will appear framework-agnostic and
 **approachable for developers coming from any background**.
 
 ### Next Steps
```

Comment:
[This section](https://deploy-preview-4186--cypress-docs.netlify.app/guides/getting-started/component-testing#Next-Steps) appears to be incomplete. At a minimum, I think we should add a link to `Learn the mount syntax`

Reply:
After further exploration, I think it actually isn't incomplete, but since it's so sparse it looks like it. Maybe we could get rid of it altogether

Reply:
This was called out in Slack too, so it might be tracked elsewhere: https://cypressio.slack.com/archives/C039JEKF9M0/p1652977210488379?thread_ts=1652976759.625619&cid=C039JEKF9M0

Reply:
Ya, this page is important as its the main landing in the docs for CT, but I agree its a bit sparse. Will probably have marketing help out with it.